suite-native/accounts/src/components/AccountsList.tsx

```diff
@@ -29,15 +33,36 @@ export const AccountsList = ({
                 DeviceRootState,
         ) => selectFilteredDeviceAccountsGroupedByNetworkAccountType(state, filterValue),
     );
+    const bottomSheetAccountAtom = useMemo(() => atom<Account | null>(null), []);
```

Comment:
Why is the atom defined inside of the component and then passed as a prop? What is the benefit of it? Could not be `useState` used in this case?

Reply:
It's actually very handy because it doesn't rerender this component when modal is opened because we use only setter here. Also another benefit is that you can use setter from child component.

Also `useMemo` is used because this component could have multiple instances across the app but we don't want to share this atom with other instances so now it basically acts as little bit smarter `useState`. It's mentioned somewhere in jotai docs that use can use it like this.